lib/SPIRV/SPIRVToOCL.cpp

```diff
@@ -576,6 +580,32 @@ void SPIRVToOCLBase::visitCallSPIRVImageMediaBlockBuiltin(CallInst *CI, Op OC) {
       },
       &Attrs);
 }
+void SPIRVToOCLBase::visitCallBuildNDRangeBuiltIn(CallInst *CI, Op OC,
+                                                  StringRef DemangledName) {
+  AttributeList Attrs = CI->getCalledFunction()->getAttributes();
+  mutateCallInstOCL(
+      M, CI,
+      [=](CallInst *Call, std::vector<Value *> &Args) {
+        auto *GWS = Args[0];
```

Comment:
NIT:
- Add an assert.
- Use more readable var names.
```suggestion
        assert(Args.size() == 3);
        auto *GlobalWorkSize = Args[0];
```

Reply:
Done, thanks